app/models/project.rb

```diff
@@ -50,6 +51,11 @@ def budget_status
     budget - hours.sum(:value) if budget
   end
 
+  def update_user_activity
+    user_activity = DateTime.now
+    save
+  end
```

Comment:
This is unnecessary, you can just use the projects `updated_at` we we have `touch: true` on the association